src/Core/Domain/CmsPage/Command/EditCmsPageCommand.php

```diff
@@ -300,7 +300,7 @@ public function getShopAssociation()
     }
 
     /**
-     * @param array|null $shopAssociation
+     * @param array $shopAssociation
```

Comment:
Nullable property => need to be solved like the others

Reply:
Fixed